### ISSUE TITLE
Bug when scheduler turned off

### ISSUE THREAD

**Issue by alexcwsmith** [LINCellularNeuroscience/VAME#26]
Hi,

When the 'scheduler' parameter is turned off, the learning rate decreases to 0 after 100 epochs. This is because in lines 432-435 of rnn_vae.py:

```
    if optimizer_scheduler:
        scheduler = StepLR(optimizer, step_size=100, gamma=0.2, last_epoch=-1)
    else:
        scheduler = StepLR(optimizer, step_size=100, gamma=0, last_epoch=-1)
```

with optimizer_scheduler deactivated, gamma should be 1, not 0. This will maintain the same learning rate throughout training. I am happy to submit a pull request if you'd like, but not sure you're currently reviewing/merging them.

In my fork I've implemented a 'manual' learning rate descent method where if scheduler=0, LR decreases if/when BEST_LOSS does not improve after STEP_SIZE epochs (and added step_size and gamma to config). If you're interested in implementing this, happy to submit a full pull request there as well, check out my fork if you want to.

Best,
Alex

**Comment by KLuxem** [comment]
Hi Alex,

thank you for being so engaging with our tool! You are absolutely right, this is a bug we need to fix. 

Cool, this sounds really like a better way of doing it! I will have a look at your fork as soon as possible and let you know.

We are currently working on some analysis parts and have hopefully time in the near future to review and merge good
pull requests.


Best,
Kevin

**Comment by KLuxem** [comment]
Closing this as with the new release we fixed the scheduler bug and also give more control over the scheduler via the config.yaml
